Paintroid/src/main/java/org/catrobat/paintroid/tools/common/Constants.kt

```diff
@@ -16,14 +16,15 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.catrobat.paintroid.tools.common
 
-package org.catrobat.paintroid.tools.common;
+class Constants {
+    companion object {
+        const val SCROLL_TOLERANCE_PERCENTAGE = 0.1f
+        const val MOVE_TOLERANCE = 5f
+    }
```

Comment:
```suggestion
const val SCROLL_TOLERANCE_PERCENTAGE = 0.1f
const val MOVE_TOLERANCE = 5f
```
Please omit the class and just write the constants.
https://kotlinlang.org/docs/properties.html#compile-time-constants

Reply:
done :)